server/src/shared/Repositories/tags.repository.ts

```diff
@@ -0,0 +1,89 @@
+// Other dependencies
+import { Repository, EntityRepository } from 'typeorm'
+
+// Local files
+import { TagsEntity } from '../Entities/tags.entity'
+
+@EntityRepository(TagsEntity)
+export class TagsRepository extends Repository<TagsEntity> {
+
+    async getTrendingTags(): Promise<{ tags: TagsEntity[], count: number }> {
+        const [tags, total] = await this.findAndCount({
+            where: {
+                updated_at: {
+                    // Last 24 hours
+                    $gte: new Date(new Date().setDate(new Date().getDate() - 1))
```

Comment:
The expression looks scary. We could create a magic number, 86400000, as a constant and use it, like:
```typescript
const DAY_IN_MS = 24 * 60 * 80 * 1000 // 86400000
```
```suggestion
                    $gte: new Date(Date.now() - DAY_IN_MS)
```